pennylane/measure.py

```diff
@@ -61,7 +61,7 @@ def circuit(x, y):
 import pennylane as qml
 
 from .qnode import QNode, QuantumFunctionError
-from .operation import Observable
```

Comment:
The CI service is picking up a cyclic import. Is it coming from this addition?

Reply:
This change shouldn't have any affect re:cyclic import, so it was likely there before.